libs/designsystem/src/lib/components/chart/options/areaspline.ts

```diff
@@ -90,7 +90,7 @@ export const AreaSplineOptions: Options = {
   series: [
     {
       type: 'areaspline',
-      data: this === undefined ? [] : this.data,
+      data: [],
```

Comment:
This doesn't seem like a safe refactoring... 🔥 
See this PR for an alternative solution: https://github.com/kirbydesign/designsystem/pull/1260